spec/models/user_spec.rb

```diff
@@ -348,14 +214,14 @@
         allow(user).to receive(:sign_in_count).and_return(1)
       end
 
-      it { is_expected.to eq(false) }
+      it { should eq(false) }
     end
   end
 
   describe "#to_analytics_json" do
-    subject{ user.to_analytics_json }
+    subject { user.to_analytics_json }
     it do
-      is_expected.to eq({
+      should eq({
```

Comment:
Before renaming `is_expected` to `should` you need to confirm with our team if this will be the default way to do one-liners.

Also, [by rspec guides](https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax), `is_expected.to` is prefered over `should`

> * **is_expected** is defined simply as expect(subject) and is designed for when you are using rspec-expectations with its newer expect-based syntax.
> * **should** was designed back when rspec-expectations only had a should-based syntax. However, it continues to be available and work even if the :should syntax is disabled (since that merely removes Object#should but this is RSpec::Core::ExampleGroup#should).